### PR TITLE
Added canIssue property to meteringpoint

### PR DESCRIPTION
**Pull request by tnickelsen**

### CODE REVIEW

File: domains/measurements/Measurements.API/API/MeteringPoints/Api/v2024_01_10/Dto/Responses/MeteringPoint.cs [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
 namespace API.MeteringPoints.Api.v2024_01_10.Dto.Responses;
 
-public record MeteringPoint(string GSRN, string GridArea, MeterType Type, SubMeterType SubMeterType, Address Address, Technology Technology)
+public record MeteringPoint(string GSRN, string GridArea, MeterType Type, SubMeterType SubMeterType, Address Address, Technology Technology, bool CanBeUsedForIssuingCertificates)
```
[the remark this answers is not in the document]
Comment (tnickelsen) [reply domains/measurements/Measurements.API/API/MeteringPoints/Api/v2024_01_10/Dto/Responses/MeteringPoint.cs:8]: We could but now we collect the logic here. It's an acceptance criteria of the story: https://app.zenhub.com/workspaces/team-atlas-633199659e255a37cd1d144f/issues/gh/energinet-datahub/energy-origin-issues/1926

File: domains/measurements/Measurements.API/Tests/MeteringPoints/Api/v2024_01_10/MeteringPointControllerTests.GetMeteringPoints.verified.txt [hunk 2]
```diff
@@ -13,7 +13,8 @@
       Technology: {
         AibTechCode: T070000,
         AibFuelCode: F00000000
-      }
+      },
+      CanBeUsedForIssuingCertificates: true
```
[the remark this answers is not in the document]
Comment (tnickelsen) [reply domains/measurements/Measurements.API/Tests/MeteringPoints/Api/v2024_01_10/MeteringPointControllerTests.GetMeteringPoints.verified.txt:17]: We can always issue certificates from consumption meteringpoints (TypeOfMp = "E17")

File: domains/measurements/Measurements.API/Tests/MeteringPoints/Api/v2024_01_10/MeteringPointControllerTests.GetMeteringPoints_GivenChildMp_ExpectChildMpOmitted.verified.txt [hunk 3]
```diff
@@ -13,7 +13,8 @@
       Technology: {
         AibTechCode: T070000,
         AibFuelCode: F00000000
-      }
+      },
+      CanBeUsedForIssuingCertificates: true
```
[the remark this answers is not in the document]
Comment (tnickelsen) [reply domains/measurements/Measurements.API/Tests/MeteringPoints/Api/v2024_01_10/MeteringPointControllerTests.GetMeteringPoints_GivenChildMp_ExpectChildMpOmitted.verified.txt:17]: We can always issue certificates from consumption meteringpoints (TypeOfMp = "E17")

File: domains/measurements/Measurements.API/Tests/Models/MeteringPointTests.cs [hunk 4]
```diff
@@ -4,57 +4,72 @@
...
+
+public class MeteringPointTests
 {
-    public class MeteringPointTests
+    [Theory]
+    [InlineData("E17", MeterType.Consumption)]
+    [InlineData("E18", MeterType.Production)]
+    [InlineData("D01", MeterType.Child)]
+    [InlineData("D99", MeterType.Child)]
+    public void MeteringPoints_MeterType_Success(string stringType, MeterType targetType)
```
[the remark this answers is not in the document]
Comment (tnickelsen) [reply domains/measurements/Measurements.API/Tests/Models/MeteringPointTests.cs:16]: Is there other types? 
We're sorting out child and other types from the response and need to convert it to child in order to do so.

File: domains/measurements/Measurements.API/Tests/SwaggerTests.GetSwaggerDocs_v20240110.verified.txt [hunk 5]
```diff
@@ -117,6 +117,9 @@
           },
           "technology": {
             "$ref": "#/components/schemas/Technology"
+          },
+          "canBeUsedForIssuingCertificates": {
+            "type": "boolean"
```
[the remark this answers is not in the document]
Comment (tnickelsen) [reply domains/measurements/Measurements.API/Tests/SwaggerTests.GetSwaggerDocs_v20240110.verified.txt:122]: Certificates subsystem will use it

File: domains/measurements/Measurements.API/API/MeteringPoints/Api/v2024_01_10/Dto/Responses/MeteringPoint.cs (hunk 1 above)
[the remark this answers is not in the document]
Comment (tnickelsen) [reply domains/measurements/Measurements.API/API/MeteringPoints/Api/v2024_01_10/Dto/Responses/MeteringPoint.cs:8]: I talked with Christoffer about this yesterday and we agreed to keep the acceptance criteria

**Review by sahma19 — APPROVED** [review]

**Pull request merged**